packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -72,14 +80,14 @@ export class DataSyncPlugin extends GraphbackPlugin {
           filter: filterTypeName
         })
 
-        // Add updatedAt arg to update and delete input types
+        // Add _version arg to update and delete input types
         // for conflict resolution
         const { fieldNames } = metadataMap;
 
         const inputType = schemaComposer.getITC(getInputTypeName(model.graphqlType.name, GraphbackOperationType.UPDATE));
-        if (inputType) {
+        if (inputType && this.conflictEngine !== undefined) {
           inputType.addFields({
-            [fieldNames.updatedAt]: {
+            [this.conflictEngine.conflictFieldName]: {
```

Comment:
It is not about field. It is method that receives object and tells you that there is conflict. 
It can be field or number of fields or hash or even call to some extra database thing etc.